database/src/main/java/database/entity/BaseEntity.java

```diff
@@ -13,4 +13,11 @@ public Integer getId() {
     public void setId(Integer id) {
         this.id = id;
     }
+
+    @Override
+    public String toString() {
```

Comment:
Let's remove all `toString` methods from entities, I don't think we need it in our application.